src/sentry/api/endpoints/organization_events_trace.py

```diff
@@ -608,6 +623,8 @@ def serialize(
         warning_extra: Dict[str, str],
         event_id: Optional[str],
         detailed: bool = False,
+        organization: Organization = None,
+        request: HttpRequest = None,
```

Comment:
Let's check the feature flag outside the serialize function and pass in the result of that flag check as something like `allow_orphan_errors` instead

Reply:
Agree.  Addressed this.